### PR TITLE
Adding Polish translation

### PR DESCRIPTION
**Pull request by Kun2210**

### CODE REVIEW

**Comment by Kun2210** [comment]
@MarshallOfSound any idea why the builds are failing? I can't find any issue with the file :O

**Comment by MarshallOfSound** [comment]
@Kun2210 You are missing a translation.

Travis CI shows the problem the clearest

```js
AssertionError: expected { Object (audio-device-communications, audio-device-default, ...) } to have a property 'settings-option-enable-api-json'
```

You simply need to add the translation key `settings-option-enable-api-json`

**Comment by Kun2210** [comment]
@MarshallOfSound Oh, didn't notice I was that behind on commits - updated, thanks for clarifying! :)

**Comment by Kun2210** [comment]
@MarshallOfSound Oh, didn't notice I was that behind on commits - updated, thanks for clarifying! :)
EDIT: I see 2 builds still failing, cannot understand if the issue has anything to do with translation file though :/

**Comment by MarshallOfSound** [comment]
Appveyor is failing on master at the moment, and I just restarted the Circle build as I believe it glitched.  Give it a sec to run.

**Comment by MarshallOfSound** [comment]
@Kun2210 Can you rebase onto master quickly, I just pushed a fixed for the CircleCI issue

**Comment by Kun2210** [comment]
Sorry, I was away for last few days, couldn't rebase it earlier - hope it's still valid :)

File: src/_locales/pl-PL.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  "settings-option-custom-theme-choose-color": "Wybierz kolor szczegółów dla własnego stylu",
+  "settings-option-custom-theme-choose-custom": "Albo wybierz",
+  "settings-option-custom-theme-custom-color": "Własny kolor",
+  "settings-option-custom-theme-dark": "Ciemny",
+  "settings-option-custom-theme-light": "Jasny",
+
+  "settings-option-enable-api": "Włącz API odtwarzania",
+  "settings-option-enable-api-json": "Włącz API JSON",
+  "settings-option-enable-api-details": "To pozwala innym aplikacjom współpracować z GPMDP",
+  "settings-option-enable-system-borders": "Użyj systemowych ramek okna aplikacji",
```
Comment (Triloworld) [inline src/_locales/pl-PL.json]: For me it isn't clear enought. Better maybe is "Użyj stylu systemowego dla okna aplikacji". It's better when there is added information about style not some system changes etc. It isn't in like "Style" section so there isn't any context what mean "systemowe ramki". It's sound like programming nature of switch, not style in Polish version.

File: src/_locales/pl-PL.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  "playback-label-volume-up": "Zwiększ głośność",
+
+  "settings-option-custom-theme": "Własny styl",
+  "settings-option-custom-theme-choose-color": "Wybierz kolor szczegółów dla własnego stylu",
+  "settings-option-custom-theme-choose-custom": "Albo wybierz",
+  "settings-option-custom-theme-custom-color": "Własny kolor",
+  "settings-option-custom-theme-dark": "Ciemny",
+  "settings-option-custom-theme-light": "Jasny",
+
+  "settings-option-enable-api": "Włącz API odtwarzania",
```
Comment (Triloworld) [inline src/_locales/pl-PL.json]: Better "Włącz dostęp poprzez API". Now "odtwarzania" is wrong translation.

File: src/_locales/pl-PL.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+  "button-test-ok": "OK",
+  "button-text-close": "Zamknij",
+  "button-text-dont-tell-me-again": "Nie pokazuj więcej",
+  "button-text-lets-do-this": "Zróbmy to!!",
+  "button-text-lets-go": "Zaczynajmy!!",
+  "button-text-not-now": "Nie teraz",
+
+  "label-about": "Informacje",
+  "label-desktop-settings": "Ustawienia Aplikacji",
```
Comment (Triloworld) [inline src/_locales/pl-PL.json]: "Ustawienia aplikacji" Capitalize first letter is not required when it isn't context of think with self name (Like name of person or name of product etc). Other application go this way when translate this word. When name of app will occur then is capitalize first letter.

File: src/_locales/pl-PL.json [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+  "message-pausing-after-song": "Włączono wstrzymanie po tym utworze.",
+  "message-pausing-after-song-button": "Wyłączono wstrzymanie po tym utworze",
+  "message-uncaught-error": "Wystąpił niespodziewany błąd w aplikacji GPMDP.",
+  "message-uncaught-error-button": "Kliknij tutaj aby zgłosić ten błąd na stronie GitHub",
+
+  "modal-confirmTray-title": "Chcieliśmy tylko uprzedzić",
+  "modal-confirmTray-content": "Zamknięcie głównego okna aplikacji (czyli to, co właśnie próbowałeś zrobić) w rzeczywistości zminimalizuje odtwarzacz do systemowego paska narzędzi. Możesz zmienić to ustawienie w oknie Ustawień Aplikacji.",
+  "modal-confirmUpdate-title": "Dostępna aktualizacja",
+  "modal-confirmUpdate-content": "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy zaktualizowanie aplikacji najszybciej jak to możliwe, żeby zapewnić najlepsze wrażenia z korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!",
```
Comment (Triloworld) [inline src/_locales/pl-PL.json]: Better: "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy aktualizacje aplikacji najszybciej jak to możliwe, aby zapewnić najlepsze wrażenia korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!". Changed "zaktualizowanie" to "aktualizacje" (it's better to prenounce).
 Change "żeby" to "aby" to remove duplicate write it. Removed "z" for the same manner.

**Comment by Triloworld** [comment]
I will add new pull request for that 👍 
Sorry for comment there. Thanks for translating it.

**Pull request merged**